static/styles/subscriptions.css

```diff
@@ -1,3 +1,5 @@
+$stream_settings_max_height: 1000px;
```

Comment:
If we’re extracting `.subscriptions-container max-height` to a variable, we should make `.subscriptions-container max-height` *use* this variable.

Reply:
It is!
```python
.user-groups-container,
.subscriptions-container {
    position: relative;
    height: 95%;
    border-radius: 4px;
    padding: 0;
    width: 97%;
    overflow: hidden;
    max-width: 1200px;
    max-height: $stream_settings_max_height;

```